lib/commit-analyzer.js

```
@@ -0,0 +1,33 @@
+const { exec } = require('child_process');
+
+const cmd = (cmd, cb) => exec(cmd, { cwd: __dirname }, (err, stdout) => cb(stdout.split('\n').join('')));
```

You're swallowing errors here :(